src/py_health/health.py

```diff
@@ -1,4 +1,29 @@
-from src.py_health import health_pb2
+# Copyright 2015 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Reference implementation for health checking in gRPC Python."""
+
+import threading
+import grpc
+
+from protos.health import health_pb2
```

Comment:
This is the one line why I'm unable to use this PR. This line HAS to be `from src.py_health import health_pb2` :/

Reply:
Why?

Reply:
The project I'm currently working on has it this way. I can't make any changes to the source files because the current build setup has it setup to work this way (One example [here](https://github.com/grpc/grpc/blob/f9f5c67aff91e4ad26371b0a2482a5011ab45226/src/python/grpcio_health_checking/grpc_health/v1/health.py)). The proto files get copied to the same directory [as a part of the setup process with `setup.py`](https://github.com/grpc/grpc/blob/f9f5c67aff91e4ad26371b0a2482a5011ab45226/src/python/grpcio_health_checking/setup.py) with the exact method to copy [here](https://github.com/grpc/grpc/blob/f9f5c67aff91e4ad26371b0a2482a5011ab45226/src/python/grpcio_health_checking/health_commands.py#L25). I need to do the same with Bazel now.